README.md

```diff
@@ -28,7 +28,7 @@ which is located in `tests/conftest.py`.
 # Setup
 Create and activate your conda environment with python3.9:
 ```commandline
-conda create -y -n substrait_consumer_testing -c conda-forge python=3.9
+conda create -y -n substrait_consumer_testing -c conda-forge python=3.9 openjdk
```

Comment:
I'll check if we need to pin the openjdk version (sometimes we need specific versions to support these java integrations)

Reply:
I did a quick check for the JPype dependency and read their setup files; I could not spot anything regarding a specific Java version.  The [documentation says it was tested with Java versions 1.7 to 13](https://jpype.readthedocs.io/en/latest/install.html), so I guess we can recommend those versions to the users without pinning.   Also, the conda package for JPype is not pinning java here ([they don't even include java as part of the dependencies in the yaml definition](https://github.com/conda-forge/jpype1-feedstock/blob/main/recipe/meta.yaml))

Reply:
Thanks for checking this!  I'l update the readme with this info.

Reply:
I believe current JPype supports 1.8 to 17.  There are scattered reports of problems wirh 19 but those appear mostly with jdk packaging issues rather than JPype itself.   JPype uses the oldest JNI [1.4] which should make it independent.  But includes a jar which uses Java 8 features and a accesses Java 9 via reflection.  In principle it should work with all versions of Java going forward.  Testing is limited to those versions on the azure cloud.